src/Examples/Akka.Hosting.SqlSharding/Program.cs

```diff
@@ -12,6 +12,53 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
+#if USE_OPTIONS
+builder.Services.AddAkka("MyActorSystem", configurationBuilder =>
+    {
+        var localConn = builder.Configuration.GetConnectionString("sqlServerLocal");
+        var shardingConn = builder.Configuration.GetConnectionString("sqlServerSharding");
+
+        var shardingJournalOptions = new SqlServerJournalOptions(isDefaultPlugin: false)
+        {
+            Identifier = "sharding",
+            ConnectionString = shardingConn,
+            AutoInitialize = true
+        };
+        var shardingSnapshotOptions = new SqlServerSnapshotOptions(isDefaultPlugin: false)
+        {
+            Identifier = "sharding",
+            ConnectionString = shardingConn,
+            AutoInitialize = true
+        };
+        
+        configurationBuilder
+            .WithRemoting("localhost", 8110)
+            .WithClustering(new ClusterOptions()
+            {
+                Roles = new[] { "myRole" },
+                SeedNodes = new[] { Address.Parse("akka.tcp://MyActorSystem@localhost:8110") }
+            })
+            .WithSqlServerPersistence(localConn)
+            .WithSqlServerPersistence(shardingJournalOptions, shardingSnapshotOptions)
```

Comment:
Do we want to log a warning or something if this happens? Or should we not worry about that for now?

Reply:
We can't really log a warning since we're composing pure HOCON configuration before the `ActorSystem` is being built... I guess we can do a `GetConfig()` and if the ID already exist we throw?